### DISCUSSION TITLE
4.5.2 (stable)

### DISCUSSION THREAD

**Discussion by PatrickAlex2019** [PatrickAlex2019/ApkEditor#5] [Announcements]
- Fixed back gesture make project close
- Fixed compile method appt, appt2
- Small changes

<hr /><em>This discussion was created from the release <a href='https://github.com/PatrickAlex2019/ApkEditor/releases/tag/4.5.2'>4.5.2 (stable)</a>.</em>

**Comment by Hiaashuu** [comment]
Please add an option to Antisplit app it's a humble request.

**Reply by jcfyre** [reply]
That is a WONDERFUL idea. I used “Antisplit” but having that all in one amazing package would be awesome!

**Reply by willianpaulista10** [reply]
Boa noite, nos galaxy s24 ultra nao estou conseguindo compilar fica dando esse erro.


![Screenshot_20250712_201541_APK Editor Pro+](https://github.com/user-attachments/assets/c6e91892-5ed8-4915-bcf1-f04485d23860)

**Comment by ahsanhabibmuaz** [comment]
Sir, any New update?

**Reply by prdx17** [reply]
![Screenshot_20250402-062325](https://github.com/user-attachments/assets/47fbe8d6-aec2-4aca-83bc-eb4f6c79b7bb)

https://www.virustotal.com/gui/file/2f9295282898e682ecd9ffeaf38f2a0b027821353c45f8219f9f185e4586cf96

**Reply by PatrickAlex2019** [reply]
The software does not contain any virus and has not been verified for modification. Anyone can use jadx or mt manager to make secondary modifications. This project will be discontinued in 2022 and there is no plan to update it for the time being. If you want to maintain or update it on this basis, you can

**Comment by metalsonic93** [comment]
When I edit an app icon it doesn't work it just shows some android icon
